src/gpo/assets/en-US/PowerToys.adml

```diff
@@ -27,6 +28,12 @@ If you enable this setting, the utility will be always enabled and the user won'
 If you disable this setting, the utility will be always disabled and the user won't be able to enable it.
 
 If you don't configure this setting, users are able to disable or enable the utility.
+</string>
+      <string id="AllowExperimentationDescription">This policy configures whether PowerToys experimentation is allowed.
+
+Disable this setting to disable the experimentation features in PowerToys, meaning the user won't see the new features being experimented even if it gets selected as part of the test group.
+
+If this setting is not configured, experimentation is allowed.
 </string>
```

Comment:
```suggestion
</string>
      <string id="AllowExperimentationDescription">This policy configures whether PowerToys experimentation is allowed. With experimentation allowed the user see the new features being experimented if it gets selected as part of the test group.

If this setting is not configured or enabled, experimentation is allowed.

If this setting is disabled, experimentation is not allowed.
 </string>
```

Reply:
Committed, but fixed some grammar issues in a following commit ;) 
Thanks for the review!